### PR TITLE
Release 0.9.6

### PR DESCRIPTION
**Pull request by AurevoirXavier**
https://github.com/darwinia-network/substrate-update-tracking/issues/4

### CODE REVIEW

File: node/service/Cargo.toml
Comment (HackFisher) [inline node/service/Cargo.toml:22]: darwinia-common does not have this branch?

File: runtime/crab/src/pallets/elections_phragmen.rs
Comment (HackFisher) [inline runtime/crab/src/pallets/elections_phragmen.rs:13]: TO Review

**Review by HackFisher — APPROVED** [review]

File: .github/workflows/ci.yml
Comment (fewensa) [inline .github/workflows/ci.yml:31]: is required? or only test use `nightly`, if is must change to `nightly` .github/workflows/release.yml need change follow this.
and this file change may conflict pr #632

Comment (AurevoirXavier) [reply .github/workflows/ci.yml:31]: Not sure the minimal support version.

But 09-20 not works.

So I set to latest nightly directly.

Comment (fewensa) [reply .github/workflows/ci.yml:31]: I think there can be add a specific nightly version, this way will help us to generate WASM have same proposal hash; and may need test `nightly-2020-10-27` is support this pr. because paritytech/srtool only have `nightly-2020-10-27`. if not support we need change use https://gitlab.com/chevdor/srtool ,  so this pr can merge first, when merged, I'll solve conflict and change CI rust toolchain version. and then build a new tag. currently need to find a suitable version.

Comment (AurevoirXavier) [reply .github/workflows/ci.yml:31]: Let's talk this while 0.8.30 coming.

File: runtime/crab/src/lib.rs
Comment (AsceticBear) [inline runtime/crab/src/lib.rs:223]: Why made this change?

Comment (AurevoirXavier) [reply runtime/crab/src/lib.rs:223]: Just because e2d go first.

**Comment by HackFisher** [comment]
At least 2 approving reviews are required before merging these PRs, please proceed for the approve or merge if there are no more changes required.
The same for other pull requests.

**Review by AsceticBear — APPROVED** [review]

**Pull request merged**
